client/src/components/profile/components/camper.css

```
@@ -6,3 +6,9 @@
   display: flex;
   justify-content: center;
 }
+
+.username,
+.name,
```

```suggestion
.name,
.location,
```
`location` can also be arbitrarily long.

---

@ojeytonwilliams Good catch. Pushing in a sec.